tests/util/iperf_proc.py

```diff
@@ -0,0 +1,196 @@
+"""
+Iperf network performance testing module.
+Used by tests for simple bandwidth measurements; uses iperf.
+
+"""
+
```

Comment:
We first wanted to simply execute ``iperf3`` and that worked fine; then we had to extend it a little further to handle ``iperf`` when ``iperf3`` proved buggy measuring udp traffic.